## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 512652`
Commit: `27a6b882d6a3cf475f3d83773161ff42b554720f`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 4 packages marked as broken and skipped:</summary>
  <ul>
    <li>kdePackages.itinerary</li>
    <li>kdePackages.itinerary.debug</li>
    <li>kdePackages.itinerary.dev</li>
    <li>kdePackages.itinerary.devtools</li>
  </ul>
</details>
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>klassy</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 127 packages built:</summary>
  <ul>
    <li>easyeffects</li>
    <li>easyeffects.debug</li>
    <li>gnome-packagekit</li>
    <li>gnome-software</li>
    <li>gnomeExtensions.easyeffects-preset-selector</li>
    <li>haruna</li>
    <li>kara</li>
    <li>kde-rounded-corners</li>
    <li>kdePackages.breeze</li>
    <li>kdePackages.breeze-gtk</li>
    <li>kdePackages.breeze-gtk.debug</li>
    <li>kdePackages.breeze-gtk.dev</li>
    <li>kdePackages.breeze-gtk.devtools</li>
    <li>kdePackages.breeze.debug</li>
    <li>kdePackages.breeze.dev</li>
    <li>kdePackages.breeze.qt5</li>
    <li>kdePackages.dolphin</li>
    <li>kdePackages.dolphin-plugins</li>
    <li>kdePackages.dolphin-plugins.debug</li>
    <li>kdePackages.dolphin-plugins.dev</li>
    <li>kdePackages.dolphin-plugins.devtools</li>
    <li>kdePackages.dolphin.debug</li>
    <li>kdePackages.dolphin.dev</li>
    <li>kdePackages.dolphin.devtools</li>
    <li>kdePackages.dynamic-workspaces</li>
    <li>kdePackages.frameworkintegration</li>
    <li>kdePackages.frameworkintegration.debug</li>
    <li>kdePackages.frameworkintegration.dev</li>
    <li>kdePackages.frameworkintegration.devtools</li>
    <li>kdePackages.karousel</li>
    <li>kdePackages.kdenetwork-filesharing</li>
    <li>kdePackages.kdenetwork-filesharing.debug</li>
    <li>kdePackages.kdenetwork-filesharing.dev</li>
    <li>kdePackages.kdenetwork-filesharing.devtools</li>
    <li>kdePackages.kdeplasma-addons</li>
    <li>kdePackages.kdeplasma-addons.debug</li>
    <li>kdePackages.kdeplasma-addons.dev</li>
    <li>kdePackages.kdeplasma-addons.devtools</li>
    <li>kdePackages.kinfocenter</li>
    <li>kdePackages.kinfocenter.debug</li>
    <li>kdePackages.kinfocenter.dev</li>
    <li>kdePackages.kinfocenter.devtools</li>
    <li>kdePackages.koi</li>
    <li>kdePackages.krohnkite</li>
    <li>kdePackages.kwin</li>
    <li>kdePackages.kwin-x11</li>
    <li>kdePackages.kwin-x11.debug</li>
    <li>kdePackages.kwin-x11.dev</li>
    <li>kdePackages.kwin-x11.devtools</li>
    <li>kdePackages.kwin.debug</li>
    <li>kdePackages.kwin.dev</li>
    <li>kdePackages.kwin.devtools</li>
    <li>kdePackages.kzones</li>
    <li>kdePackages.oxygen</li>
    <li>kdePackages.oxygen.debug</li>
    <li>kdePackages.oxygen.dev</li>
    <li>kdePackages.oxygen.qt5</li>
    <li>kdePackages.packagekit-qt (qt6Packages.packagekit-qt)</li>
    <li>kdePackages.plasma-browser-integration</li>
    <li>kdePackages.plasma-browser-integration.debug</li>
    <li>kdePackages.plasma-browser-integration.dev</li>
    <li>kdePackages.plasma-browser-integration.devtools</li>
    <li>kdePackages.plasma-desktop</li>
    <li>kdePackages.plasma-desktop.debug</li>
    <li>kdePackages.plasma-desktop.dev</li>
    <li>kdePackages.plasma-desktop.devtools</li>
    <li>kdePackages.plasma-integration</li>
    <li>kdePackages.plasma-integration.debug</li>
    <li>kdePackages.plasma-integration.dev</li>
    <li>kdePackages.plasma-integration.qt5</li>
    <li>kdePackages.plasma-login-manager</li>
    <li>kdePackages.plasma-login-manager.debug</li>
    <li>kdePackages.plasma-login-manager.dev</li>
    <li>kdePackages.plasma-login-manager.devtools</li>
    <li>kdePackages.plasma-mobile</li>
    <li>kdePackages.plasma-mobile.debug</li>
    <li>kdePackages.plasma-mobile.dev</li>
    <li>kdePackages.plasma-mobile.devtools</li>
    <li>kdePackages.plasma-pa</li>
    <li>kdePackages.plasma-pa.debug</li>
    <li>kdePackages.plasma-pa.dev</li>
    <li>kdePackages.plasma-pa.devtools</li>
    <li>kdePackages.plasma-setup</li>
    <li>kdePackages.plasma-setup.debug</li>
    <li>kdePackages.plasma-setup.dev</li>
    <li>kdePackages.plasma-setup.devtools</li>
    <li>kdePackages.plasma-welcome</li>
    <li>kdePackages.plasma-welcome.debug</li>
    <li>kdePackages.plasma-welcome.dev</li>
    <li>kdePackages.plasma-welcome.devtools</li>
    <li>kdePackages.plasma-workspace</li>
    <li>kdePackages.plasma-workspace.debug</li>
    <li>kdePackages.plasma-workspace.dev</li>
    <li>kdePackages.plasma-workspace.devtools</li>
    <li>kdePackages.plasma-workspace.sessions</li>
    <li>kdePackages.powerdevil</li>
    <li>kdePackages.powerdevil.debug</li>
    <li>kdePackages.powerdevil.dev</li>
    <li>kdePackages.powerdevil.devtools</li>
    <li>kdePackages.sierra-breeze-enhanced (qt6Packages.sierra-breeze-enhanced)</li>
    <li>kdePackages.xdg-desktop-portal-kde</li>
    <li>kdePackages.xdg-desktop-portal-kde.debug</li>
    <li>kdePackages.xdg-desktop-portal-kde.dev</li>
    <li>kdePackages.xdg-desktop-portal-kde.devtools</li>
    <li>nordic</li>
    <li>nordic.sddm</li>
    <li>packagekit</li>
    <li>packagekit.dev</li>
    <li>packagekit.devdoc</li>
    <li>pantheon.elementary-greeter</li>
    <li>pantheon.elementary-onboarding</li>
    <li>pantheon.elementary-settings-daemon</li>
    <li>pantheon.switchboard-plug-about</li>
    <li>pantheon.switchboard-plug-keyboard</li>
    <li>pantheon.switchboard-plug-pantheon-shell</li>
    <li>pantheon.switchboard-plug-power</li>
    <li>pantheon.switchboard-plug-security-privacy</li>
    <li>pantheon.switchboard-with-plugs</li>
    <li>pantheon.wingpanel-applications-menu</li>
    <li>pantheon.wingpanel-indicator-power</li>
    <li>pantheon.wingpanel-quick-settings</li>
    <li>pantheon.wingpanel-with-indicators</li>
    <li>plasma-panel-colorizer</li>
    <li>plasma-panel-spacer-extended</li>
    <li>simple-scan</li>
    <li>system-config-printer</li>
    <li>utterly-nord-plasma</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>klassy</summary>
<pre>/nix/store/qd70v8g0561vm8m33kmnp79z00cgyi5n-gcc-wrapper-15.2.0/bin/g++ -DKCOREADDONS_LIB -DKLASSY_GIT_MASTER=0 -DKLASSY_VERSION=\&quot;6.5.3\&quot; -DKPLUGINFACTORY_PLUGIN_CLASS_INTERNAL_NAME=klassystyleconfig_factory -DQT_CORE_LIB -DQT_DBUS_LIB -DQT_GUI_LIB -DQT_NETWORK_LIB -DQT_NO_DEBUG -DQT_QMLINTEGRATION_LIB -DQT_QML_LIB -DQT_WIDGETS_LIB -DTRANSLATION_DOMAIN=\&quot;klassy_style_config\&quot; -D_GNU_SOURCE -D_LARGEFILE64_SOURCE -Dklassystyleconfig_EXPORTS -I/build/source/build/kstyle6/config -I/build/source/kstyle/config -I/build/source/build/kstyle6/config/klassystyleconfig_autogen/include -I/build/source/kstyle/animations -I/build/source/kstyle/debug -I/build/source/libbreezecommon -I/build/source/build/libbreezecommon6 -isystem /nix/store/1q8sx67miwfn3ws5k7mkmkcjbym4akkp-qtbase-6.11.0/include/QtCore -isystem /nix/store/1q8sx67miwfn3ws5k7mkmkcjbym4akkp-qtbase-6.11.0/mkspecs/linux-g++ -isystem /nix/store/1q8sx67miwfn3ws5k7mkmkcjbym4akkp-qtbase-6.11.0/include/QtGui -isystem /nix/store/1q8sx67miwfn3ws5k7mkmkcjbym4akkp-qtbase-6.11.0/include/QtWidgets -isystem /nix/store/1q8sx67miwfn3ws5k7mkmkcjbym4akkp-qtbase-6.11.0/include/QtDBus -isystem /nix/store/80i8cngma2i9bvmklw1fw0qgwy66pgsd-kcoreaddons-6.25.0-dev/include/KF6/KCoreAddons -isystem /nix/store/31hyig8cppbfg60vl5gslifvdfz7fc1c-kconfig-6.25.0-dev/include/KF6/KConfig -isystem /nix/store/31hyig8cppbfg60vl5gslifvdfz7fc1c-kconfig-6.25.0-dev/include/KF6/KConfigCore -isystem /nix/store/nqd0hjnzhiwiq1ldsb8pxyypahhz27p0-kconfigwidgets-6.25.0-dev/include/KF6/KConfigWidgets -isystem /nix/store/4d9fmz8glh4zdniwag3wfc93l9s42f64-kwidgetsaddons-6.25.0-dev/include/KF6/KWidgetsAddons -isystem /nix/store/31hyig8cppbfg60vl5gslifvdfz7fc1c-kconfig-6.25.0-dev/include/KF6/KConfigGui -isystem /nix/store/xl0yxmg5133gplghgaj0d2wvhms8id13-kcolorscheme-6.25.0-dev/include/KF6/KColorScheme -isystem /nix/store/4qm49jn18vrm8fzabyz248nnn871dk2y-ki18n-6.25.0-dev/include/KF6/KI18n -isystem /nix/store/y61mcazkhmxblibvzcb6hs128z7rmd2f-kcmutils-6.25.0-dev/include/KF6/KCMUtils -isystem /nix/store/y61mcazkhmxblibvzcb6hs128z7rmd2f-kcmutils-6.25.0-dev/include/KF6/KCMUtilsQuick -isystem /nix/store/lwa1k7ni8d4ljj44mzzlagk4qnngrsr8-qtdeclarative-6.11.0/include/QtQml -isystem /nix/store/lwa1k7ni8d4ljj44mzzlagk4qnngrsr8-qtdeclarative-6.11.0/include/QtQmlIntegration -isystem /nix/store/1q8sx67miwfn3ws5k7mkmkcjbym4akkp-qtbase-6.11.0/include/QtNetwork -isystem /nix/store/y61mcazkhmxblibvzcb6hs128z7rmd2f-kcmutils-6.25.0-dev/include/KF6/KCMUtilsCore -fno-operator-names -fno-exceptions -Wall -Wextra -Wcast-align -Wchar-subscripts -Wformat-security -Wno-long-long -Wpointer-arith -Wundef -Wnon-virtual-dtor -Woverloaded-virtual -Werror=return-type -Werror=init-self -Wvla -Wdate-time -Wsuggest-override -Wlogical-op -fdiagnostics-color=always -O3 -DNDEBUG -std=gnu++20 -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -fPIC -MD -MT kstyle6/config/CMakeFiles/klassystyleconfig.dir/klassystyleconfig_autogen/mocs_compilation.cpp.o -MF kstyle6/config/CMakeFiles/klassystyleconfig.dir/klassystyleconfig_autogen/mocs_compilation.cpp.o.d -o kstyle6/config/CMakeFiles/klassystyleconfig.dir/klassystyleconfig_autogen/mocs_compilation.cpp.o -c /build/source/build/kstyle6/config/klassystyleconfig_autogen/mocs_compilation.cpp
In file included from /build/source/build/kstyle6/config/klassystyleconfig_autogen/EWIEGA46WW/../../../../../kstyle/config/breezestyleconfig.h:10,
                 from /build/source/build/kstyle6/config/klassystyleconfig_autogen/EWIEGA46WW/moc_breezestyleconfig.cpp:9,
                 from /build/source/build/kstyle6/config/klassystyleconfig_autogen/mocs_compilation.cpp:2:
/build/source/libbreezecommon/breeze.h:15:10: fatal error: breezesettings.h: No such file or directory
   15 | #include &quot;breezesettings.h&quot;
      |          ^~~~~~~~~~~~~~~~~~
compilation terminated.
[12/138] Building CXX object kstyle6/config/CMakeFiles/klassystyleconfig.dir/breezestyleconfig.cpp.o
FAILED: [code=1] kstyle6/config/CMakeFiles/klassystyleconfig.dir/breezestyleconfig.cpp.o 
/nix/store/qd70v8g0561vm8m33kmnp79z00cgyi5n-gcc-wrapper-15.2.0/bin/g++ -DKCOREADDONS_LIB -DKLASSY_GIT_MASTER=0 -DKLASSY_VERSION=\&quot;6.5.3\&quot; -DKPLUGINFACTORY_PLUGIN_CLASS_INTERNAL_NAME=klassystyleconfig_factory -DQT_CORE_LIB -DQT_DBUS_LIB -DQT_GUI_LIB -DQT_NETWORK_LIB -DQT_NO_DEBUG -DQT_QMLINTEGRATION_LIB -DQT_QML_LIB -DQT_WIDGETS_LIB -DTRANSLATION_DOMAIN=\&quot;klassy_style_config\&quot; -D_GNU_SOURCE -D_LARGEFILE64_SOURCE -Dklassystyleconfig_EXPORTS -I/build/source/build/kstyle6/config -I/build/source/kstyle/config -I/build/source/build/kstyle6/config/klassystyleconfig_autogen/include -I/build/source/kstyle/animations -I/build/source/kstyle/debug -I/build/source/libbreezecommon -I/build/source/build/libbreezecommon6 -isystem /nix/store/1q8sx67miwfn3ws5k7mkmkcjbym4akkp-qtbase-6.11.0/include/QtCore -isystem /nix/store/1q8sx67miwfn3ws5k7mkmkcjbym4akkp-qtbase-6.11.0/mkspecs/linux-g++ -isystem /nix/store/1q8sx67miwfn3ws5k7mkmkcjbym4akkp-qtbase-6.11.0/include/QtGui -isystem /nix/store/1q8sx67miwfn3ws5k7mkmkcjbym4akkp-qtbase-6.11.0/include/QtWidgets -isystem /nix/store/1q8sx67miwfn3ws5k7mkmkcjbym4akkp-qtbase-6.11.0/include/QtDBus -isystem /nix/store/80i8cngma2i9bvmklw1fw0qgwy66pgsd-kcoreaddons-6.25.0-dev/include/KF6/KCoreAddons -isystem /nix/store/31hyig8cppbfg60vl5gslifvdfz7fc1c-kconfig-6.25.0-dev/include/KF6/KConfig -isystem /nix/store/31hyig8cppbfg60vl5gslifvdfz7fc1c-kconfig-6.25.0-dev/include/KF6/KConfigCore -isystem /nix/store/nqd0hjnzhiwiq1ldsb8pxyypahhz27p0-kconfigwidgets-6.25.0-dev/include/KF6/KConfigWidgets -isystem /nix/store/4d9fmz8glh4zdniwag3wfc93l9s42f64-kwidgetsaddons-6.25.0-dev/include/KF6/KWidgetsAddons -isystem /nix/store/31hyig8cppbfg60vl5gslifvdfz7fc1c-kconfig-6.25.0-dev/include/KF6/KConfigGui -isystem /nix/store/xl0yxmg5133gplghgaj0d2wvhms8id13-kcolorscheme-6.25.0-dev/include/KF6/KColorScheme -isystem /nix/store/4qm49jn18vrm8fzabyz248nnn871dk2y-ki18n-6.25.0-dev/include/KF6/KI18n -isystem /nix/store/y61mcazkhmxblibvzcb6hs128z7rmd2f-kcmutils-6.25.0-dev/include/KF6/KCMUtils -isystem /nix/store/y61mcazkhmxblibvzcb6hs128z7rmd2f-kcmutils-6.25.0-dev/include/KF6/KCMUtilsQuick -isystem /nix/store/lwa1k7ni8d4ljj44mzzlagk4qnngrsr8-qtdeclarative-6.11.0/include/QtQml -isystem /nix/store/lwa1k7ni8d4ljj44mzzlagk4qnngrsr8-qtdeclarative-6.11.0/include/QtQmlIntegration -isystem /nix/store/1q8sx67miwfn3ws5k7mkmkcjbym4akkp-qtbase-6.11.0/include/QtNetwork -isystem /nix/store/y61mcazkhmxblibvzcb6hs128z7rmd2f-kcmutils-6.25.0-dev/include/KF6/KCMUtilsCore -fno-operator-names -fno-exceptions -Wall -Wextra -Wcast-align -Wchar-subscripts -Wformat-security -Wno-long-long -Wpointer-arith -Wundef -Wnon-virtual-dtor -Woverloaded-virtual -Werror=return-type -Werror=init-self -Wvla -Wdate-time -Wsuggest-override -Wlogical-op -fdiagnostics-color=always -O3 -DNDEBUG -std=gnu++20 -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -fPIC -MD -MT kstyle6/config/CMakeFiles/klassystyleconfig.dir/breezestyleconfig.cpp.o -MF kstyle6/config/CMakeFiles/klassystyleconfig.dir/breezestyleconfig.cpp.o.d -o kstyle6/config/CMakeFiles/klassystyleconfig.dir/breezestyleconfig.cpp.o -c /build/source/kstyle/config/breezestyleconfig.cpp
In file included from /build/source/kstyle/config/breezestyleconfig.h:10,
                 from /build/source/kstyle/config/breezestyleconfig.cpp:8:
/build/source/libbreezecommon/breeze.h:15:10: fatal error: breezesettings.h: No such file or directory
   15 | #include &quot;breezesettings.h&quot;
      |          ^~~~~~~~~~~~~~~~~~
compilation terminated.
[13/138] Building CXX object kstyle6/config/CMakeFiles/klassystyleconfig.dir/klassystyleconfig_autogen/UYX5XTB5RZ/qrc_klassystyleconfig.cpp.o
[14/138] Building CXX object kstyle6/config/CMakeFiles/klassystyleconfig.dir/breezestyleconfigdata.cpp.o
ninja: build stopped: subcommand failed.</pre>
</details>
</details>

--- a/pkgs/tools/package-management/packagekit/default.nix
+++ b/pkgs/tools/package-management/packagekit/default.nix
@@ -10,6 +10,8 @@
   sqlite,
   gobject-introspection,
   vala,
+  jansson,
+  docbook_xsl_ns,
   gtk-doc,
   boost,
   meson,
@@ -30,7 +32,7 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "packagekit";
-  version = "1.3.3";
+  version = "1.3.5";
 
   outputs = [
     "out"
@@ -42,7 +44,7 @@ stdenv.mkDerivation (finalAttrs: {
     owner = "PackageKit";
     repo = "PackageKit";
     rev = "v${finalAttrs.version}";
-    hash = "sha256-BgVfM2EtuvV9qTFSy+WW5Ny1QrHIj3t2Royrn7ZHAA8=";
+    hash = "sha256-aKucwqwNyZWyHfNu9ntzSwD+eQy8KjCt6RVMjjjZmZg=";
   };
 
   buildInputs = [
@@ -52,6 +54,7 @@ stdenv.mkDerivation (finalAttrs: {
     gst_all_1.gstreamer
     gst_all_1.gst-plugins-base
     gtk3
+    jansson
     sqlite
     boost
   ]
@@ -97,6 +100,9 @@ stdenv.mkDerivation (finalAttrs: {
       --replace-fail "install_dir: join_paths(get_option('sysconfdir'), 'PackageKit')" "install_dir: join_paths('$out', 'etc', 'PackageKit')"
     substituteInPlace data/meson.build \
       --replace-fail "install_dir: join_paths(get_option('localstatedir'), 'lib', 'PackageKit')," "install_dir: join_paths('$out', 'var', 'lib', 'PackageKit'),"
+    substituteInPlace client/meson.build \
+      --replace-fail http://docbook.sourceforge.net/release/xsl-ns/current ${docbook_xsl_ns}/share/xml/docbook-xsl-ns
+
   '';
 
   passthru.tests = {

